### PR TITLE
[13.0][MIG] auth_from_http_remote_user

### PR DESCRIPTION
**Pull request by apineux**
Migration to version 13.0 of module auth_from_http_remote_user

### CODE REVIEW

File: auth_from_http_remote_user/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Author: Laurent Mignon
+# Copyright 2014-2018 'ACSONE SA/NV'
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    'name': 'Authenticate via HTTP Remote User',
+    'version': '11.0.1.0.0',
```
Comment (ThomasBinsfeld) [inline auth_from_http_remote_user/__manifest__.py]:
```suggestion
    'version': '13.0.1.0.0',
```

File: auth_from_http_remote_user/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Author: Laurent Mignon
+# Copyright 2014-2018 'ACSONE SA/NV'
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
```
Comment (ThomasBinsfeld) [inline auth_from_http_remote_user/__manifest__.py]:
```suggestion
# Copyright 2014-2018 'ACSONE SA/NV'
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
```
(idem for all files)

File: auth_from_http_remote_user/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+{
+    'name': 'Authenticate via HTTP Remote User',
+    'version': '11.0.1.0.0',
+    'category': 'Tools',
```
Comment (ThomasBinsfeld) [inline auth_from_http_remote_user/__manifest__.py]: IMO it's worth a stable [development status key](https://odoo-community.org/page/development-status).
```suggestion
    'category': 'Tools',
    'development_status': "Production/Stable",
```

File: auth_from_http_remote_user/models/res_users.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        readonly=True,
+        copy=False
+    )
+
+    def _check_credentials(self, password):
+        """Check credentials for SSO user"""
+        res = self.sudo().search([('id', '=', self._uid),
+                                  ('sso_key', '=', password)])
+        if not res:
+            return super(Users, self)._check_credentials(password)
```
Comment (ThomasBinsfeld) [inline auth_from_http_remote_user/models/res_users.py]:
```suggestion
            return super()._check_credentials(password)
```

File: auth_from_http_remote_user/controllers/main.py [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    _REMOTE_USER_ATTRIBUTE = 'HTTP_REMOTE_USER'
+
+    @http.route('/web', type='http', auth="none")
+    def web_client(self, s_action=None, **kw):
+        main.ensure_db()
+        try:
+            self._bind_http_remote_user(http.request.session.db)
+        except http.AuthenticationError:
+            return werkzeug.exceptions.Unauthorized().get_response()
+        return super(Home, self).web_client(s_action, **kw)
```
Comment (ThomasBinsfeld) [inline auth_from_http_remote_user/controllers/main.py]:
```suggestion
        return super().web_client(s_action, **kw)
```

**Review by ThomasBinsfeld — COMMENTED** [review]
Only minor comments. Could you run pre-commit to make Travis green?

**Review by ThomasBinsfeld — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**